gloo/docs/gloo_routing/virtual_services/security/custom_auth/_index.md

```diff
@@ -192,25 +238,29 @@ spec:
           upstream:
             name: default-petstore-8080
             namespace: gloo-system
+EOF
 {{< /highlight >}}
 
-To make it easy, if you have followed this guide verbatim, you can just download and apply [this](gloo-vs.yaml) manifest to update both Settings and VirtualService.
+If you have followed this guide verbatim, you can just download and apply [this manifest](gloo-vs.yaml)  to update 
+both the `Settings` and the `Virtual Service`.
 
 ## Test
 
-We are all set to test!
+Let's verify that our configuration has been accepted by Gloo. Requests to `/api/pets/1` should be allowed:
 
 ```shell
-curl --write-out "%{http_code}\n" "$(glooctl proxy url)/api/pets/1"
+curl --write-out "%{http_code}\n" $GATEWAY_URL/api/pets/1
```

Comment:
I'd honestly like to use `$(glooctl proxy url)` everywhere and drop `GATEWAY_URL`, it doesn't seem more convenient to me

Reply:
the command is just as easy since its copy-pastable, and its one less thing to copy

Reply:
further, its always up to date. you don't have to worry about any past exported `GATEWAY_URL`s

Reply:
It comes from a template that we use in a bunch of places. If we change it, I'd . rather do it in a separate PR.